spotify.go

```diff
@@ -217,7 +218,14 @@ func retryDuration(resp *http.Response) time.Duration {
 
 func (c *Client) get(url string, result interface{}) error {
 	for {
-		resp, err := c.http.Get(url)
+		req, err := http.NewRequest("GET", url, nil)
+		if c.AcceptLanguage != "" {
+			req.Header.Set("Accept-Language", c.AcceptLanguage)
```

Comment:
Should we also set this header in `execute()`?

Reply:
I missed it. Added.